### PR TITLE
[New] Add driver.js@0.5.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #12686

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/kamranahmedse/driver.js
 * Official website (optional, not the repository): http://kamranahmed.info/driver
 * NPM package url (optional): https://www.npmjs.com/package/driver.js
 * License and its reference: MIT, https://github.com/kamranahmedse/driver.js/blob/master/license
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 169
   - Count of stars: 7763
   - Count of forks: 301
 * NPM download stats (optional):
   - Downloads in the last day: 66
   - Downloads in the last week: 375
   - Downloads in the last month: 1632

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
Use npm auto-update for more versions.

File: ajax/libs/driver.js/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "driver.js",
+  "filename": "driver.min.js",
+  "version": "0.5.0",
+  "description": "A light-weight, no-dependency, vanilla JavaScript library to drive the user's focus across the page",
+  "homepage": "http://kamranahmed.info/driver",
```
Comment (sufuf3) [inline ajax/libs/driver.js/package.json]: Let's use `https`
What do you think?
Many thanks

Comment (PeterDaveHello) [reply ajax/libs/driver.js/package.json]: Nice catch ：+1:

**Comment by extend1994** [comment]
@sufuf3 Updated, please help check it, thanks!

**Pull request merged**